frontends/web/src/routes/lightning/send/send.tsx

```diff
@@ -252,6 +252,7 @@ export const Send = () => {
   const [customAmount, setCustomAmount] = useState<number>();
   const [rawInputError, setRawInputError] = useState<string>();
   const [sendError, setSendError] = useState<string>();
+  const [useTrampoline] = useState<boolean>(true);
```

Comment:
Why use a state here?

Reply:
I added it in case you wanted to switch between trampoline/non-trampoline in the app.

Reply:
I'd remove it for now, since we're not using it. We can always put it back :) Thanks :pray: